pages/api/xata-global.ts

```diff
@@ -46,6 +46,9 @@ export default async function api(req: Request) {
   for (let i = 0; i < count; i++) {
     data = await xata.db.employees
       .select(["emp_no", "first_name", "last_name"])
+      .filter({
+        last_name: {$startsWith: String.fromCharCode(65 + i)}
+      })
```

Comment:
should we revert this for now? we can send it as a separate PR.

Reply:
Oh, yeah, will revert this.